programs/eosio-launcher/main.cpp

```diff
@@ -1244,6 +1244,18 @@ launcher_def::write_logging_config_file(tn_node_def &node) {
    if( gelf_enabled ) ttf.appenders.push_back( "net" );
    log_config.loggers.emplace_back( ttf );
 
+  fc::logger_config ttst( "transient_trx_success_tracing" );
+  ttst.level = ll;
+  ttst.appenders.push_back( "stderr" );
+  if( gelf_enabled ) ttst.appenders.push_back( "net" );
+  log_config.loggers.emplace_back( ttst );
+
+  fc::logger_config ttft( "transient_trx_failure_tracing" );
+  ttft.level = ll;
+  ttft.appenders.push_back( "stderr" );
+  if( gelf_enabled ) ttft.appenders.push_back( "net" );
+  log_config.loggers.emplace_back( ttft );
```

Comment:
`eosio-launcher` is being removed by #192. 
@jgiszczak can you add these to the python replacement for this. It is ok to do that even before this PR is merged because unreferenced loggers are ignored.

Reply:
I did that for running the tests.

Reply:
Jonathan is going to add this to the new python regression test suite, so you don't need to worry about it.